notebooks/post_DL3_analysis.ipynb

```diff
@@ -0,0 +1,2633 @@
+{
```

Comment:
<p>i am confused, what is the status of the storage of RAD_MAX?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/955/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Current master of lstchain stores it correctly as float in degrees, the version of the school stored the string `"0.2 deg". The notebook should reflect only the current state, not what was before.</p>